core/src/banking_stage/unprocessed_transaction_storage.rs

```diff
@@ -35,18 +32,6 @@ pub const UNPROCESSED_BUFFER_STEP_SIZE: usize = 64;
 /// Maximum number of votes a single receive call will accept
 const MAX_NUM_VOTES_RECEIVE: usize = 10_000;
 
-#[derive(Debug)]
-pub enum UnprocessedTransactionStorage {
-    VoteStorage(VoteStorage),
-    LocalTransactionStorage(ThreadLocalUnprocessedPackets),
-}
-
-#[derive(Debug)]
-pub struct ThreadLocalUnprocessedPackets {
-    unprocessed_packet_batches: UnprocessedPacketBatches,
-    thread_type: ThreadType,
-}
```

Comment:
cannot comment because too far from the diff - `ThreadType` enum no longer needs to exist

Reply:
afaict `PacketBatchInsertionMetrics` also does not need to exist anymore; and same for `InsertPacketBatchSummary` since we only need `VoteBatchInsertionMetrics` now, right?